src/platform/silabs/SilabsConfig.cpp

```diff
@@ -284,6 +284,11 @@ CHIP_ERROR SilabsConfig::WriteConfigValueStr(Key key, const char * str, size_t s
 
     if (str != NULL)
```

Comment:
Can we use the verify or return logic here as well?

Reply:
Yes but I didn't want to go through the whole exit cleanup in our original SilabsConfig since the goal is to switch to SilabsConfigCTM (And remove CTM) once the Common Token Manager is in a useable state.

The cleanup of SilabsConfig is a task in itself.